src/main/java/org/gridsuite/gateway/filters/UserAdminControlGlobalPreFilter.java

```diff
@@ -30,10 +30,8 @@
 public class UserAdminControlGlobalPreFilter extends AbstractGlobalPreFilter {
     private static final Logger LOGGER = LoggerFactory.getLogger(UserAdminControlGlobalPreFilter.class);
 
-    private UserAdminService userAdminService;
-
     public UserAdminControlGlobalPreFilter(UserAdminService userAdminService) {
-        this.userAdminService = userAdminService;
+        super(userAdminService);
```

Comment:
Add a comment for this class saying that this comes after other filters (getOrder), and we don't use it to reject request, only to record connection. Optionally remove the dead code that rejects (clientList and else block at the end) or add a comment saying it can't happen and needs to be refactored